app/models/katello/content_view_history.rb

```diff
@@ -15,12 +15,15 @@ class ContentViewHistory < Katello::Model
     validates_lengths_from_database
     validates :status, :inclusion => {:in          => STATUSES,
                                       :allow_blank => false}
+    validates :action, presence: true, on: :create
```

Comment:
If you look at the migration, it's possible that if the task for the content view doesn't exist (meaning it's been cleaned up) and since we don't know what type of history it is without the task, it's possible that action may be blank for pre-existing histories. So therefore, I figured we should only validate its presence on create.

I guess another alternative would be to validate it on create and validate that it's not being unset on update. I can do that as well. Also, histories can't be updated currently I don't think so we could theoretically just always validate them to.